src/SFA.DAS.Campaign.Api.UnitTests/Controllers/TrainingCourses/WhenGettingStandardsBySector.cs

```diff
@@ -37,13 +38,14 @@ public class WhenGettingStandardsBySector
             controllerResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
             var model = controllerResult.Value as GetStandardsResponse;
             Assert.IsNotNull(model);
-            model.Standards.Should().BeEquivalentTo(mediatorResult.Standards, options => options);
+            model.Standards.Should().BeEquivalentTo(mediatorResult.Standards, options => options.ExcludingMissingMembers());
```

Comment:
`ExcludingMissingMembers` is a good utility, however I think we should exclusively list members so when new members are added, the test fails and gives developer an indication of impact, against which they can take conscious actions.

Reply:
I've switched to specifics on `StandardUId` and `LarsCode` I don't like either, but at least as you say this is explicit.